tools/extra/mmlink/mmlink_connection.cpp

```diff
@@ -198,7 +201,7 @@ int mmlink_connection::handle_unbound_command(char *cmd)
 				" setting to bound state\n";
 
 		bind();
-		send(OK, strnlen_s(OK, sizeof(OK)));
+		send(OK, strnlen_s(OK, MMLINK_OK_SIZE));
```

Comment:
Seeing as this is C++ already, can't you use C++ strings instead?

Reply:
extra size for "\0" , added +1 to it